example/MODULE.bazel

```diff
@@ -1,7 +1,7 @@
 "Bazel dependencies"
 
 bazel_dep(name = "aspect_rules_lint", version = "0.0.0")
-bazel_dep(name = "aspect_bazel_lib", version = "2.5.0")
+bazel_dep(name = "aspect_bazel_lib", version = "2.6.1")
```

Comment:
If you're bumping to latest then 2.7.0 is latest atm

Reply:
Ah, I should have commented. This bump is minimal needed to fix the GLIBC version error I was getting on BCR releases.